portfolio/src/main/webapp/index.html

```diff
@@ -32,6 +33,11 @@ <h2>BACKGROUND</h2>
           <p>After taking extracurricular courses to create 2D games using unity, I realized how powerful and vast this field could be so I started taking online courses 
               for programming in C# and Python on my own time. This helped me decide to enroll in Computer Science.</p>
           <p>My goal is to be able to create elaborate creative experiences using technology!</p>
+
+          <button onclick="addFunction();">Greetings</button>
+          <button onclick="getMarcoStats();">Marco</button>
```

Comment:
Check out [`addEventListener()`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) as a way to manage your page's functionality strictly from your JavaScript instead of using `onclick`. You might also need to add the [`defer`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer) attribute to your script tag to make this work properly.